backend/src/main/java/com/bakdata/conquery/sql/conversion/model/select/SqlSelectId.java

```diff
@@ -0,0 +1,25 @@
+package com.bakdata.conquery.sql.conversion.model.select;
+
+import com.bakdata.conquery.models.datasets.concepts.Concept;
+import com.bakdata.conquery.models.datasets.concepts.select.Select;
+import com.bakdata.conquery.models.types.ResultType;
+import lombok.Value;
+
+@Value
+public class SqlSelectId {
```

Comment:
das bitte nicht Id nennen, das hat in Conquery eine ziemlich spezifische/definierte Semantik